main/options/options.cc

```diff
@@ -1101,7 +1110,8 @@ void readOptions(Options &opts,
         }
         if (raw["license"].as<bool>()) {
             fmt::print(
-                "Sorbet typechecker is licensed under Apache License Version 2.0.\n\nSorbet is built on top of:\n{}",
+                "Sorbet typechecker is licensed under Apache License Version 2.0.\n\nSorbet is built on top "
+                "of:\n{}",
```

Comment:
Interesting, I wonder why this changed? I guess maybe an errant clang-format while the file was malformed?